spec/config/mongoid.yml

```diff
@@ -3,7 +3,9 @@ test:
     default:
       database: mongoid_test
       hosts:
-        - <%=ENV["MONGOID_SPEC_HOST"]%>:<%=ENV["MONGOID_SPEC_PORT"]%>
+        <% SpecConfig.instance.addresses.each do |address| %>
```

Comment:
Given that these changes don't seem related, I think a rebase might be necessary?

Reply:
Agreed 👍. These were not my changes; will revert all the unrelated test environment changes.